src/styles/calendar.scss

```diff
@@ -0,0 +1,72 @@
+@use 'sass:list';
+
+.page {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  &__body {
+    margin: 0;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
```

Comment:
The variables $gap and $padding are used here, but they are not defined in this file. Make sure these variables are defined in a file that is imported or in this file itself.